python-package/pyproject.toml

```diff
@@ -82,6 +80,13 @@ experimental = false
 strict-config = false
 minimum-version = "build-system.requires"
 
+[tool.scikit-build.cmake]
+version = "CMakeLists.txt"
+build-type = "Release"
+
+[tool.scikit-build.cmake.define]
+__BUILD_FOR_PYTHON = "ON"
+
```

Comment:
What is the benefit of moving these things down here? Are the equivalent fields in `[tool.scikit-build]` deprecated?

Reply:
Huh, these nested fields are identical, I simply don't know how to specify nested mapping `tool.scikit-build.cmake.define` in existing setup 😬

Reply:
I agree with you about that, I'd expected that this form would work:

```toml
[tool.scikit-build]
cmake.define = [
  "__BUILD_FOR_PYTHON=ON",
  "some_new_option=value"
]
```

But when I search around on GitHub, I don't see any examples like that: https://github.com/search?q=%22cmake.define%22+language%3ATOML&type=code&p=1

Let's leave this as you have it in the PR, I like it.